extensions/markdown-language-features/package.json

```diff
@@ -121,6 +121,14 @@
       }
     ],
     "commands": [
+      {
+        "command": "markdown.copyImage",
+        "title": "Copy",
```

Comment:
Maybe `Copy image` instead?

Also this title needs to be localizable. Take a look at how the other commands do this. Basically you create a new entry in `package.nls.json` with `"mykey": "Copy Image"` and then reference that key here by wrapping it in `%`: `%mykey%`